content/applications/finance/fiscal_localizations/austria.rst

```diff
@@ -0,0 +1,137 @@
+=======
+Austria
+=======
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the Austrian
+localization.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+     - License
+   * - :guilabel:`Austria - Accounting`
+     - `l10n_at`
+     - Default :ref:`fiscal localization package <fiscal_localizations/packages>`.
+     - Community
+   * - :guilabel:`Austria - Accounting Reports`
+     - `l10n_at_reports`
+     - Adds localized versions of financial reports
+     - Enterprise
+   * - :guilabel:`Austrian SAF-T Export`
+     - `l10n_at_saft`
+     - Adds the SAF-T export.
+     - Enterprise
+
+.. image:: austria/austria-modules.png
+   :alt: Modules for the Austrian localization
+
+Financial Reports
+=================
+
+.. seealso::
+   :doc:`../accounting/reporting`
+
+Currently the following localized reports are available:
+
+  - Balance sheet according to § 224 UGB
+  - Profit and loss according to § 231 UGB (Gesamtkostenverfahren)
+
+To populate the individual lines of these reports the mapping from
+your company's chart of accounts to the SAF-T accounts is used.
+See :ref:`the information about the chart of accounts below <austria/SAFT/chart_of_accounts>`
+
+Standard Audit File for Tax (SAF-T)
+===================================
+
+The Austrian tax office may request a SAF-T report on a on-demand basis.
+The :guilabel:`Austrian SAF-T Export` module provides the ability to export a SAF-T document
+(in XML format).
+
+Configuration
+-------------
+
+The following explains how to ensure that all the information that is needed
+for the SAF-T export is available. In case missing information is detected during the export process
+a warning message will appear that lists the missing information.
+
+Company
+~~~~~~~
+
+Under :guilabel:`Settings --> General Settings`, in the **Companies** section, click
+:guilabel:`Update Info` and fill in the following information:
+
+- Fill in the :guilabel:`Address` and :guilabel:`Phone` fields.
+  For the address please enter (at least) :guilabel:`Street`, :guilabel:`City`, :guilabel:`ZIP`
+  and :guilabel:`Country`.
+- Provide your company's tax ID in the :guilabel:`Company ID` field.
+- If you have one, provide your :abbr:`UID-Nummer (Umsatzsteueridentifikationsnummer)`
+  in the :guilabel:`Tax ID` field number, including the country prefix.
+
+You must have at least one **contact person** linked to your company in the **Contacts** app:
```

Comment:
You must have at least one **contact person** linked to your company in the :guilabel:`Contacts` app: